cmd/did-method-cli/createconfigcmd/createconfig.go

```diff
@@ -189,9 +191,16 @@ func getConfig(cmd *cobra.Command) (*config, error) {
 	}
 
 	for _, member := range config.MembersData {
-		if err := member.jsonWebKey.UnmarshalJSON(member.PrivateKeyJwk); err != nil {
-			return nil, err
+		jwkData, err := ioutil.ReadFile(member.PrivateKeyJwkPath) //nolint: gosec
```

Comment:
i hope we are not assuming all keys are available on a system.

some stakeholders will sign their own copy on their own system.

Reply:
i.e. this is somewhat like fabric endorsement - ask the stakeholder to sign it.

Reply:
@Moopli https://github.com/trustbloc/trustbloc-did-method/issues/150